Resources/Prototypes/Recipes/Reactions/chemicals.yml

```diff
@@ -412,6 +408,49 @@
   products:
     SodiumCarbonate: 4
 
+- type: reaction
+  id: Benzene
+  impact: Medium
+  reactants:
+    Hydrogen: 
+      Amount: 1
+    Carbon:
+      Amount: 1
+  products:
+    Benzene: 0.5
+
+- type: reaction
+  minTemp: 310 # Emo, you could change this to use one of your fancy machines, provided that machine is roundstart
+  id: Hydroxide
+  reactants:
+    Oxygen:
+      Amount: 1
+    Hydrogen:
+      Amount: 1
+  products:
+    Hydroxide: 2
+
+- type: reaction
+  id: SodiumHydroxide
+  impact: Medium
```

Comment:
If this is a normal precursor, then this should not have a medium impact.

Reply:
Same here